File: Hm2/2_v3.py
import os
import sys

def main():
    fd = open('TextFileV2.txt', 'r')
    spaces = 0
    for spaces,line in enumerate(fd):
        spaces += line.count(' ')
        print('Spaces %d' % (spaces))
    fd.close()
if __name__ == '__main__':
    main()

File: Hm2/2_v3.c
#include<stdio.h>
int main(){
    char ss[255];
    FILE *spaces;
    spaces = fopen("textFileV2.txt","r");
    fgets (ss, 255, spaces);
    fclose(spaces);
    int ch, spacenum = 0;
    for(ch=0;ch<255;ch++)
        if(ss[ch] == ' ')
            spacenum++;
    printf("%d Spaces:", spacenum);
    getchar();
    return 0;
}
